### PR TITLE
Add Cyberwatch detection format

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: Cyberwatch/cyberwatch-detection/_meta/fields.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+cyberwatch.active:
```
Comment (Adamowoc) [inline Cyberwatch/cyberwatch-detection/_meta/fields.yml:1]:
```suggestion
cyberwatch.vas.active:
```
Based on `cyberwatch.vas.fixed_at`. What do you think @squioc ?

Comment (squioc) [reply Cyberwatch/cyberwatch-detection/_meta/fields.yml:1]: Yes. I agree with you. I think all custom fields should be prefixed with `cyberwatch.vas`.

**Review by Adamowoc — APPROVED** [review]
LGTM

**Comment by penhouetp** [comment]
@Adamowoc @lvoloshyn-sekoia Thanks for the job 💪 Can we deploy it in production asap? A customer is waiting for this integration since many month

**Pull request merged**